numba/tests/doc_examples/test_structref_usage.py

```diff
@@ -0,0 +1,149 @@
+# "magictoken" is used for markers as beginning and ending of example text.
+
+import unittest
+
+# magictoken.ex_structref_type_definition.begin
+import numpy
```

Comment:
```suggestion
import numpy as np
```
and use `np` throughout